### PR TITLE
snake_game

### PR DESCRIPTION
**Pull request by Anmol-Sharma21**
# Pull Request 

**Feature Name:** Snake Game

**Changes made:**<br>

1) removed the redundant files/folders:
- .idea/
- .pycache/

2) removed the DS_store files

### Self Check(Tick After Making pull Request)

- [x] I have read and followed the [Contribution Guidelines](/CONTRIBUTING.md)
- [x] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in the proper folder

- [x] PR is regarding Code
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [ ] I am using external libraries and I have created requirements.txt in my project folder
- [x] Script is tested and running perfectly fine on my system

- [ ] PR is regarding the documentation

> If the issue was not assigned to you Please don't make a PR, It will be marked as invalid.

Discuss in discord server:- http://pragmaticprogrammer.in/discord

## acknowledgment
- [x] I have tested my code and distributing it under GPL-3.0 License

### CODE REVIEW

File: Projects/snake_game/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Snake Game using Python
+
+## Description
+This is a classic Snake game implemented in Python using the Tkinter library for GUI. The game allows the player to control a snake which moves around the screen, eating food pellets and growing longer. The objective is to avoid colliding with the walls or the snake's own body while trying to eat as much food as possible to score points.
```
Comment (Punit-Choudhary) [inline Projects/snake_game/README.md:4]: You mentioned Tkinter, while project seems to be using turtle. If this is a typo, please fix.

**Review by Punit-Choudhary — CHANGES_REQUESTED** [review]
Please remove these redundant files/folders:
- .idea/
- __pycache__/

**Comment by Anmol-Sharma21** [comment]
made the needful changes in the file

**Review by Punit-Choudhary — CHANGES_REQUESTED** [review]
Remove these unnecessary files:
- .DS_Store
- Project/.DS_Store

Please also lint your code using flake8 or any online code beautifier.

**Comment by Anmol-Sharma21** [comment]
removed the DS_store files

File: Projects/snake_game/data.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+15
```
Comment (Punit-Choudhary) [inline Projects/snake_game/data.txt:1]: Reset high score to 0

File: Projects/snake_game/scoreboard.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    def reset(self):
+        if self.score > self.high_score:
+            self.high_score = self.score
+            with open("data.txt", mode="w") as data:
+                data.write(f"{self.high_score}")
+        self.score = 0
+        self.update_scorebaord()
+
+    # def game_over(self):
```
Comment (Punit-Choudhary) [inline Projects/snake_game/scoreboard.py:31]: Remove unused code

**Review by Punit-Choudhary — CHANGES_REQUESTED** [review]
Please lint your code using flake8 or any online code beutifier.

**Comment by Punit-Choudhary** [comment]
Please resolve these linting issues:

./Projects/snake_game/food.py:19:38: W292 no newline at end of file
./Projects/snake_game/main.py:35:80: E501 line too long (114 > 79 characters)
./Projects/snake_game/main.py:47:1: E303 too many blank lines (4)
./Projects/snake_game/scoreboard.py:16:80: E501 line too long (102 > 79 characters)
./Projects/snake_game/scoreboard.py:21:80: E501 line too long (102 > 79 characters)
./Projects/snake_game/snake.py:1:1: F403 'from turtle import *' used; unable to detect undefined names
./Projects/snake_game/snake.py:23:5: E303 too many blank lines (2)
./Projects/snake_game/snake.py:24:19: F405 'Turtle' may be undefined, or defined from star imports: turtle
./Projects/snake_game/snake.py:31:5: E303 too many blank lines (2)
./Projects/snake_game/snake.py:[43](https://github.com/Punit-Choudhary/Python-beginner-scripts/actions/runs/8482667203/job/23261066916?pr=275#step:4:44):5: E303 too many blank lines (3)

**Comment by Punit-Choudhary** [comment]
./Projects/snake_game/main.py:35:80: E501 line too long (114 > 79 characters)
./Projects/snake_game/snake.py:1:1: F403 'from turtle import *' used; unable to detect undefined names
./Projects/snake_game/snake.py:10:1: E302 expected 2 blank lines, found 1
./Projects/snake_game/snake.py:22:19: F405 'Turtle' may be undefined, or defined from star imports: turtle

Please lint your code and resolve these.
Let me know if you want any help resolving these.

**Review by Punit-Choudhary — APPROVED** [review]

**Comment by Punit-Choudhary** [comment]
Thank you for the contribution 💜 🤝

**Pull request merged**